[kitties-tutorial] File not found for module `mock` and `tests`

https://docs.substrate.io/tutorials/v3/kitties/pt1/#set-a-price-for-each-kitty
This helper file in the tutorial is wrong.
```
error[E0583]: file not found for module `mock`
 --> pallets\kitties\src\lib.rs:5:1
  |
5 | mod mock;
  | ^^^^^^^^^
  |
  = help: to create the module `mock`, create file "pallets\kitties\src\mock.rs" or "pallets\kitties\src\mock\mod.rs"

error[E0583]: file not found for module `tests`
 --> pallets\kitties\src\lib.rs:6:1
  |
6 | mod tests;
  | ^^^^^^^^^^
  |
  = help: to create the module `tests`, create file "pallets\kitties\src\tests.rs" or "pallets\kitties\src\tests\mod.rs"
```

--- a/static/assets/tutorials/kitties-tutorial/04-interacting-functions.rs
+++ b/static/assets/tutorials/kitties-tutorial/04-interacting-functions.rs
@@ -2,9 +2,6 @@
 
 pub use pallet::*;
 
-mod mock;
-mod tests;
-
 #[frame_support::pallet]
 pub mod pallet {
   use frame_support::pallet_prelude::*;

